src/qtfirebaseadmob.cpp

```diff
@@ -1123,8 +1132,10 @@ firebase::FutureBase QtFirebaseAdMobRewardedVideoAd::initInternal()
     return firebase::FutureBase();
 }
 
-void QtFirebaseAdMobRewardedVideoAd::onPresentationStateChanged(int state)
+void QtFirebaseAdMobRewardedVideoAd::setPresentationState(QtFirebaseAdMobInterstitial::PresentationState state)
 {
+    emit presentationStateChanged(state);
+
```

Comment:
For some reason I didn't get [the warning](https://github.com/Larpon/QtFirebase/pull/62/files#diff-7c49a17150f2b1d0ec90f86a32dc1471R1007) mentioned above here?